dds_web/api/project.py

```diff
@@ -486,8 +485,11 @@ def delete_project(self, project: models.Project, current_time: datetime.datetim
             )
```

Comment:
Side note: I started looking at delete_project_contents which then led me to the `remove_bucket_contents` in `api_s3_connector.py`. I checked the `bucket.objects.all().delete()` documentation and it says this regarding `delete()`

> The request can contain a list of up to 1,000 keys that you want to delete.

Since `all()` can definitely return more than 1000 objects in some cases (although the risk isn't as likely atm since most are zipping / taring the files) we need to add a new task in the backlog for improving this so that it can handle any number of objects. Also, I'm currently not sure what happens if the number of objects is more than 1000, and if this could potentially be causing the timeouts / issues in general as well.

Reply:
Possibly adding https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/bucket/objects.html#page_size would do

Reply:
https://scilifelab.atlassian.net/browse/HMS-2396